drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -363,9 +383,6 @@ local function do_configure(driver, device)
         profile_name = profile_name .. "-cooling-only"
       end
 
-      -- TODO remove this in favor of reading Thermostat clusters AttributeList attribute
```

Comment:
Is this no longer needed? This should be left if we still need to do it, which I think we do

Reply:
There is another place that states the TODO, it looked kinda clunky having it written twice (even if the change applies to both)